### PR TITLE
Fix python3 part2

### PR DESCRIPTION
**Pull request by aalex**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 ===================
 
-Version 0.8.0 - 2016-??-??
+Version 0.8.0 - 2020-08-??
 --------------------------
-This release is an official version in the 0.8 stable series. That means its interfaces and API should be compatible with every release in this series.
-
-New features: 
-* (todo)
+This release contains a lot of fixes and improvements.
```
Comment (vgrangep) [inline CHANGELOG.md:6]: I suggest you had that we are also dropping the support of python 2.7

**Review by vgrangep — COMMENTED** [review]
I would just update the changelog
For the rest, LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (aalex) [reply CHANGELOG.md:6]: Can you approve it?

Comment (aalex) [reply CHANGELOG.md:6]: @vgrangep I usually update the changelog when we do a release. (not right now)

**Review by baydam — APPROVED** [review]

**Pull request merged**